client/state/selectors/test/is-gutenberg-opt-in-dialog-showing.js

```diff
@@ -0,0 +1,43 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
```

Comment:
Let's use jest over `chai`.  (eg you don't need this import to use `expect` some of the BDD notation is slightly different. For example toBe vs to.be

https://jestjs.io/docs/en/expect